<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPrealertEdit</class>
 <widget class="QWidget" name="CPrealertEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="cmbPersonal"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QDateEdit" name="datFromDate">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="displayFormat">
          <string>dd.MM.yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCalendarWidget" name="calDate">
         <property name="gridVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="trvDutyTypes">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="cmdNewDuty">
           <property name="text">
            <string>^ Hin&amp;zufügen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdKillDuty">
           <property name="text">
            <string>L&amp;öschen</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>10</number>
     </property>
     <property name="midLineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdNew">
       <property name="text">
        <string>&amp;Neu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdKill">
       <property name="text">
        <string>Lös&amp;chen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="trvPrealerts">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbPersonal</tabstop>
  <tabstop>datFromDate</tabstop>
  <tabstop>cmdNew</tabstop>
  <tabstop>cmdKill</tabstop>
  <tabstop>trvPrealerts</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
